tools/misbehaving-jmx-server/src/main/java/org/datadog/misbehavingjmxserver/App.java

```diff
@@ -82,14 +156,16 @@ public static void main( String[] args ) throws IOException, MalformedObjectName
             jCommander.usage();
             System.exit(1);
         }
+        //get bean count and properties from config file
+        config.readConfigFileOnDisk();
 
         InterruptibleRMISocketFactory customRMISocketFactory = new InterruptibleRMISocketFactory();
         // I don't think this call is actually important for jmx, the below 'env' param to JMXConnectorServerFactory is the important one
         RMISocketFactory.setSocketFactory(customRMISocketFactory);
 
-        // Explicitly set RMI hostname to specified argument value
-        System.setProperty("java.rmi.server.hostname", config.rmiHost);
-        
+        // Uncomment this to be able to access jmx server on MacOS
```

Comment:
lets not make this change as part of this PR.
This is what I'd call a "drive-by" change. The overall intention of this PR is to enable more configurable beans to be exposed and allow it to be driven by a config file on disk.  
This change is not related to this PR's intention and this change would benefit from further description, hence why I recommend to remove it.